datasets/gov/migris/prieglobscio_suteikimas.csv

```diff
@@ -0,0 +1,22 @@
+﻿id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/migris/prieglobscio_suteikimas,,,,,,,,,,,,,"Duomenys apie prieglobsčio prašytojus, prieglobstį gavusius užsieniečius, pateiktus prašymus suteikti prieglobstį ir priimtus sprendimus, duomenys apie užsieniečius, dėl kurių buvo išsiųsti ar gauti paklausimai (ir/ar atsakymai) dėl jų perėmimo savo žinion ar atsiėmimo pagal Dublino III reglamentą, duomenys apie užsieniečių perdavimą pagal Dublino III reglamentą."
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,PrieglobscioPrasytojas,,,id,,,3,open,,Duomenys apie prieglobsčio prašytojus,
+,,,,,id,string,,,,4,open,,Unikalus identifikatorius,
+,,,,,prasymo_registravimo_data,date,D,,,4,open,,Prašymo prieglobsčiui gauti registracijos data,

```

Comment:
VDV IS yra pateiktas toks `property=prasymo_pateikimo_data`, publikuojant duomenis Saugykloje pavadinimai turėtų sutapti.

Reply:
Ačiū, pataisiau stulpelio pavadinimą Palantire.